pkg/query/stat_processor.go

```diff
@@ -174,7 +180,7 @@ func (sp *defaultStatProcessor) process(workers uint) {
 		_, _ = fmt.Printf("Saving High Dynamic Range (HDR) Histogram of Response Latencies to %s\n", sp.args.hdrLatenciesFile)
 		var b bytes.Buffer
 		bw := bufio.NewWriter(&b)
-		_, err = statMapping[allQueriesLabel].latencyHDRHistogram.PercentilesPrint(bw, 10, 1000.0)
+		_, err = sp.statMapping[labelAllQueries].latencyHDRHistogram.PercentilesPrint(bw, 10, 1000.0)
```

Comment:
Whats the reason for switch from `allQueriesLabel` const? It seems to be used in the code of the function prior to this line.

Reply:
typo. addressed in latest commit